e2e/signer.service.e2e.ts

```diff
@@ -4,7 +4,7 @@ import { ProviderType, SignerService } from '../src';
 describe('Signer tests', () => {
   it('should set `isEthrSigner` for node environment during initialization', async () => {
     const wallet = Wallet.createRandom();
-    const provider = new providers.JsonRpcProvider('http://localhost:8544');
+    const provider = new providers.JsonRpcProvider('http://127.0.0.1:8544');
```

Comment:
I suppose this was the main fix? Changing from `localhost` to `127.0.0.1`?

Reply:
Yes, that was the reason